RecoLocalCalo/HGCalRecProducers/src/classes_def.xml

```diff
@@ -0,0 +1,5 @@
+<lcgdict>
+  <class name="std::map<DetId,const HGCRecHit*>" persistent="false"/>
+  <class name="edm::Wrapper<std::map<DetId,const HGCRecHit*>>" persistent="false"/>
+  <class name="std::pair<const DetId,const HGCRecHit*>" />
+</lcgdict>
```

Comment:
If I understand (well, guess actually) the meaning of the `lostDefs` complaing, these dictionaries should be defined in the package where `HGCRecHit` is defined, i.e. in `DataFormats/HGCRecHit`.